docs/migration-guide.rst

```diff
@@ -197,6 +197,31 @@ handler, you must call :py:func:`girder.events.unbind` on the existing mapping f
     # Prints 'a' and 'b' in Girder 2, but only 'a' in Girder 3
     events.trigger('an_event')
 
+Removed or moved plugins
+++++++++++++++++++++++++
+
+Many plugins were either deleted from the main repository, or moved to other repositories. Plugins
+that were moved to other repositories will no longer be installed via the ``[plugins]`` extra when
+installing the ``girder`` python package, but can be installed by
+``pip install girder_plugin_[name]``. If you were depending on a plugin that was deleted
```

Comment:
should that be `pip install girder-plugin-[name]` ? this would be more consistent with python convention